src/components/React/OpacityControl.js

```diff
@@ -1,12 +1,9 @@
 import React, { Component } from "react";
+import { Button } from "react-bootstrap";
 import { connect } from "react-redux";
 import { mySendAlphaPoints } from "../../redux/AppActions";
 
-const mapStateToProps = (state) => {
-  return { volumeData: state.volumeData };
-};
-
-export default connect(mapStateToProps, { mySendAlphaPoints })(
```

Comment:
is this the transfer function component? if so, can we rename it?

Reply:
I think so? More renaming in #38